api/src/writers.py

```diff
@@ -15,18 +15,27 @@ def write(self, value):
 
 
 class CSVExport:
-    """Class to (download) an iterator to a 
+    """Class to (download) an iterator to a
     CSV file."""
 
-    def export(self, filename, iterator, serializer, header):
+    def export(self, filename, iterator, serializer=None, header=None, streaming=False):
         # 1. Create our writer object with the pseudo buffer
         writer = csv.writer(CSVBuffer())
 
+        if not header:
+            header = iterator[0].keys()
+
+        if not serializer:
+            serializer = lambda x: x.values()
+
         # 2. Create the HttpResponse using our iterator as content
-        response = HttpResponse(
+        cls = HttpResponse
+        if streaming:
+            cls = StreamingHttpResponse
```

Comment:
Ik ben zelf meer fan van de one-liner, maar dat is wel echt heel erg personal preference: 

`cls = StreamingHttpResponse if streaming else HttpReponse`